isotovideo

```diff
@@ -87,15 +87,15 @@ sub usage {
 
 sub version {
     my $thisversion = qx{git rev-parse HEAD};
```

Comment:
Just for future reference, this line is modified by a spec file when we're packaging: https://build.opensuse.org/package/view_file/devel:openQA/os-autoinst/os-autoinst.spec

Reply:
@foursixnine THanks! I didn't knew that...